protocols/imap/src/main/java/org/apache/james/imap/processor/base/SelectedMailboxImpl.java

```diff
@@ -162,8 +161,8 @@ public synchronized int recentCount() {
     }
 
     @Override
-    public synchronized MailboxPath getPath() {
-        return path;
+    public synchronized MailboxPath getPath() throws MailboxException {
```

Comment:
does it still require to be `synchronized` ?

Reply:
No